### PR TITLE
List

### PR DESCRIPTION
**Pull request by sidra2022**
Add some changes to the List. Please Review this according to your capacity.

### CODE REVIEW

File: packages/stencil/src/components/datacom-list/datacom-list.css [hunk 1]
```diff
@@ -3,38 +3,29 @@
...
   list-style: disc outside none;
 }
 
-.list-lowercase {
+.dc-list-lowercase {
   list-style: lower-alpha;
 }
 
-.list-upercase {
+.dc-list-upercase {
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-list/datacom-list.css:25]: Misspelt 'upercase'. Should be 'uppercase'.

File: apps/storybook/src/stories/datacom-list.stories.tsx [hunk 2]
```diff
@@ -18,58 +16,196 @@ export default {
...
     type: {
       name: 'Type',
-      control: 'select',
-      defaultValue: 'numbers',
       description: 'List type  is default  numbers if not set',
       options: ['numbers', 'lowercase', 'upercase', 'roman'],
     },
+    heading: {
+      name: 'Heading',
+      description: 'For heading style of list inside paragrap',
```
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-list.stories.tsx:24]: Misspelt 'paragrap'. Should be 'paragraph'.

File: packages/stencil/src/components/datacom-list/datacom-li.css [hunk 3]
```diff
@@ -1,36 +1,30 @@
 :host {
   display: block;
+
+  --li-padding: 8px;
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-list/datacom-li.css:4]: Considering you're not reusing this value in multiple places in this file, you probably don't need to make this a variable. But there's no harm in doing so, so feel free to leave this if you want.

File: packages/stencil/src/components/datacom-list/datacom-list.css (hunk 1 above)
Comment (AngeCast) [reply packages/stencil/src/components/datacom-list/datacom-list.css:25]: Do remember to also update your usage.md file and your stories with the same spelling correction above.

File: packages/stencil/src/components/datacom-list/usage.md [hunk 4]
```diff
@@ -1,20 +1,48 @@
-# list Useage
-
-```html
+# list Usage
 
+-List composed of two components : 'datacom-list' and 'datacom-li'.
+-List have two main variant ordered and unordered.
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-list/usage.md:4]: For better clarity, could write this sentence as:
List has two main variants: 'ordered' and 'unordered'.

**Review by AngeCast — CHANGES_REQUESTED** [review]

File: packages/stencil/src/components/datacom-list/datacom-list.css (hunk 1 above)
Comment (sidra2022) [reply packages/stencil/src/components/datacom-list/datacom-list.css:25]: Done

File: apps/storybook/src/stories/datacom-list.stories.tsx (hunk 2 above)
Comment (sidra2022) [reply apps/storybook/src/stories/datacom-list.stories.tsx:24]: Fixed.

File: packages/stencil/src/components/datacom-list/usage.md (hunk 4 above)
Comment (sidra2022) [reply packages/stencil/src/components/datacom-list/usage.md:4]: Fixed

File: packages/stencil/src/components.d.ts [hunk 5]
```diff
@@ -254,9 +253,15 @@ export namespace Components {
         "value"?: string;
     }
     interface DatacomLi {
-        "variantItem": ItemStyle;
```
Comment (AngeCast) [inline packages/stencil/src/components.d.ts:257]: Mind reverting the changes to this file? It's auto-generated.

**Review by AngeCast — CHANGES_REQUESTED** [review]
Just one last comment from me.

File: packages/stencil/src/components.d.ts (hunk 5 above)
Comment (sidra2022) [reply packages/stencil/src/components.d.ts:257]: Fixed.

**Review by AngeCast — APPROVED** [review]

**Pull request merged**